mem0/memory/main.py

```diff
@@ -646,14 +645,17 @@ def history(self, memory_id):
         capture_event("mem0.history", self, {"memory_id": memory_id, "sync_type": "sync"})
         return self.db.get_history(memory_id)
 
-    def _create_memory(self, data, existing_embeddings, metadata=None):
+    def _create_memory(self, data, existing_embeddings, metadata: dict | None = None):
```

Comment:
Change this to `Optional[dict]` because this project supports Python >=3.9 and `|` was introduced in Python 3.10